phpBB/styles/prosilver/template/login_body.html

```diff
@@ -27,6 +27,7 @@ <h2 class="login-title"><!-- IF LOGIN_EXPLAIN -->{LOGIN_EXPLAIN}<!-- ELSE -->{L_
 		<!-- ENDIF -->
 		<!-- IF S_DISPLAY_FULL_LOGIN -->
 		<dl>
+			<dt>&nbsp;</dt>
 			<!-- IF S_AUTOLOGIN_ENABLED --><dd><label for="autologin"><input type="checkbox" name="autologin" id="autologin" tabindex="4" /> {L_LOG_ME_IN}</label></dd><!-- ENDIF -->
 			<dd><label for="viewonline"><input type="checkbox" name="viewonline" id="viewonline" tabindex="5" /> {L_HIDE_ME}</label></dd>
```

Comment:
For each `dd` a `dt` should be placed above it, AFAIR.

Reply:
No, there can be one dt with many dd elements.

Reply:
See also the specs: https://www.w3.org/TR/2011/WD-html5-author-20110809/the-dl-element.html